src/main/resources/templates/snowflake/Makefile.schema.ssp

```diff
@@ -14,11 +14,24 @@ See the License for the specific language governing permissions and
 limitations under the License.
 --%>
 <%@ val tables: Seq[io.phdata.streamliner.schemadefiner.model.TableDefinition] %>
-<%
-	val firstRunTables = tables.map(t => s"first-run-${t.destinationName}").mkString(" ")
-	val runTables = tables.map(t => s"run-${t.destinationName}").mkString(" ")
-	val cleanTables = tables.map(t => s"clean-${t.destinationName}").mkString(" ")
-%>
+<%@ val tableDiffs: Seq[io.phdata.streamliner.schemadefiner.model.TableDiff] %>
+#if (!tableDiffs.isEmpty)
+    <%
```

Comment:
Does tableDiffs take care of the case where a new table is created inside the source database? If not, how will that case be handled?

Reply:
I believe creating a new table in source db is handled. Let me test it once. I will update here

Reply:
This case is handled.  I created a new table STREAMLINER_TEST2 in oracle and then performed schema evolution step.
![Screenshot 2021-07-13 at 7 34 48 PM](https://user-images.githubusercontent.com/83769498/125466118-2d5be7e2-88c5-4c8a-8315-5914f795129d.png)